src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/HourlyWageCalculator.cs

```diff
@@ -22,10 +22,10 @@ public decimal Calculate(decimal wageValue, WageUnit wageUnit, decimal hoursPerW
                     calculatedHourlyWage = decimal.Divide(wageValue, hoursPerWeek);
                     break;
                 case WageUnit.Monthly:
-                    calculatedHourlyWage = decimal.Divide(decimal.Divide(wageValue, WeeksPerMonth), hoursPerWeek);
+                    calculatedHourlyWage = decimal.Divide(decimal.Multiply(wageValue, MonthsPerYear), decimal.Multiply(WeeksPerYear, hoursPerWeek));
```

Comment:
This stuff sends me a bit loopy. I am guessing this is to align with what happens in RAA Wages.cs file HourlyWage calculation method but with less and maybe nicer code.

Reply:
Correct